frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -122,6 +127,13 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({ state, hasOCS
               })}
             </ListItem>
           )}
+          {isTaintSupported && (
+            <ListItem>
```

Comment:
This looks like an old issue. But each of these list items should have a key.

Reply:
dynamically generated list need keys so react identify them. We dont need that here.